src/components/EachCharacterCard/EachCharacterCard.js

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+
+import "./EachCharacterCard.scss";
+
+function EachCharacterCard({ name, image, species, status, origin, location }) {
```

Comment:
It is just an opinion, but this component's name doesn't seem very descriptive to me, cause it has the same name than 'CharacterCard' which is also for 'each' character, so I suggest a name like 'CharacterProfileSheet' or something.